internal/gatewayapi/testdata/grpcroute-with-valid-ratelimitfilter.out.yaml

```diff
@@ -0,0 +1,124 @@
+gateways:
```

Comment:
should be tested out using https://gateway.envoyproxy.io/v0.5.0/user/grpc-routing.html and https://gateway.envoyproxy.io/v0.5.0/user/rate-limit.html